Mathlib/Lean/Expr/Basic.lean

```diff
@@ -115,17 +115,6 @@ def isBlackListed {m} [Monad m] [MonadEnv m] (declName : Name) : m Bool := do
 
 end Name
 
-namespace NameSet
-
-/-- The union of two `NameSet`s. -/
-def append (s t : NameSet) : NameSet :=
-  s.mergeBy (fun _ _ _ => .unit) t
-
-instance : Append NameSet where
-  append := NameSet.append
-
-end NameSet
-
```

Comment:
You can ignore the changes to this file. They are needed here to get Mathlib to compile against this toolchain, but do not need to be reviewed.